README.md

```diff
@@ -132,17 +133,14 @@ class App extends Component {
     return (
       <DragDropContext onDragEnd={this.onDragEnd}>
         <Droppable droppableId="droppable">
-          {(provided, snapshot) => (
+          {(provided, snapshot) =>
             <div
               ref={provided.innerRef}
               style={getListStyle(snapshot.isDraggingOver)}
             >
-              {this.state.items.map(item => (
```

Comment:
when a function inside a render method returns JSX the convention for this codebase is that the returned value is wrapped inside parentheses `( <MyComponent /> )`. I think it helps to read big trees. If you could please update all the examples to go back to using parentheses that would be great